ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -674,6 +674,58 @@ def verify_kms_ca_only():
         )
 
 
+def add_capacity_test():
+    from ocs_ci.ocs.cluster import (
+        is_flexible_scaling_enabled,
+        check_ceph_health_after_add_capacity,
+    )
+
+    osd_size = get_osd_size()
+    existing_osd_pods = get_osd_pods()
+    existing_osd_pod_names = [pod.name for pod in existing_osd_pods]
+    result = add_capacity(osd_size)
+    osd_pods_post_expansion = get_osd_pods()
+    osd_pod_names_post_expansion = [pod.name for pod in osd_pods_post_expansion]
```

Comment:
instead of osd_pods_post_expansion you can use get_osd_pods() if osd_pods_post_expansion is not used further in the code